oracle.py

```diff
@@ -32,11 +33,12 @@ def update_legality(self, legal_cards):
 
     def update_database(self, new_version):
         self.database.execute('DELETE FROM version')
-        self.database.execute('DELETE FROM card')
```

Comment:
If mtgjson has an update, but our database doesn't, then we go straight into this without dropping the tables.